test_runner/regress/test_lsn_mapping.py

```diff
@@ -107,6 +111,17 @@ def test_lsn_mapping(neon_env_builder: NeonEnvBuilder):
         # make sure that we return the minimum lsn here at the start of the range
         assert Lsn(result["lsn"]) >= last_flush_lsn
 
+        client.configure_failpoints((failpoint, "return"))
```

Comment:
Better to add some comments that link to this pull request, or have a separate test case testing the behavior of cancellation. In general, I feel it's weird to mix cancellation test into a test case called `test_lsn_mapping`.

Reply:
Thanks for the catch. I forgot to actually raise the exception on Python side. I will also create a separate test for this as well.

Reply:
test case failed because the failpoint is not hit on the vector path. Moved the failpoint to a common path that also return `ReconstructError::Cancelled`.